### DISCUSSION TITLE
packRGB not working

### DISCUSSION THREAD

**Discussion by MintiestKitKat** [cc-tweaked/CC-Tweaked#1210] [Q&A]
i'm just getting into this mod, and i'm having an issue already: packRBG doesn't appear to be working properly.
![grafik](https://user-images.githubusercontent.com/77462121/200187531-c58b7b8c-fc33-49c7-9016-ff5000dbd23e.png)
![grafik](https://user-images.githubusercontent.com/77462121/200187532-871f54b8-93b5-4220-82ee-def5cf8513f6.png)
what is happening here?

**Comment by Wojbie** [comment]
paintutils drawPixel function expects one of those colours: https://tweaked.cc/module/colors.html

If you want to use your own color you need to use palette to change one of 16 colours to be yours.. For example change lighBlue to be whatever colour you are trying to use here with palettes and then put lightBlue into drawPixel.
